src/components/Details/Details.js

```diff
@@ -1,75 +1,77 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import './Details.css';
 import DetailsCardList from './DetailsCardList';
 import WriteReview from './WriteReview';
 import Album from './Album';
 import songData from '../TestData/songData.json'
 import TrackList from './TrackList';
-import albumData from '../TestData/albumData.json';
 import { useParams } from 'react-router';
 import testReviews from '../TestData/testReviews.json'
+import { getAlbumBySpotifyId } from '../../services/album-service';
 
 function Details() {
-    const [artists, setArtists] = useState('');
-    const [foundAlbum, setFoundAlbum] = useState(false);
-    const [album, setAlbum] = useState(albumData);
     const { albumID } = useParams();
+    const [artists, setArtists] = useState('');
+    const [album, setAlbum] = useState();
+    const [shouldFetch, setShouldFetch] = useState(true);
 
     const formatArtistNames = (artists) => {
-        if (artists.length === 0) {
-          return '';
-        } else if (artists.length === 1) {
-          return artists[0].name;
-        } else if (artists.length === 2) {
-          return `${artists[0].name} and ${artists[1].name}`;
-        } else {
-          return artists.reduce((acc, curr, index) => {
-            if (index === artists.length - 1) {
-              return `${acc} and ${curr.name}`;
-            } else {
-              return `${acc}, ${curr.name}`;
-            }
-          }, '');
-        }
+      if (artists.length === 0) {
+        return '';
+      } else if (artists.length === 1) {
+        return artists[0].name;
+      } else if (artists.length === 2) {
+        return `${artists[0].name} and ${artists[1].name}`;
+      } else {
+        return artists.reduce((acc, curr, index) => {
+          if (index === artists.length - 1) {
+            return `${acc} and ${curr.name}`;
+          } else {
+            return `${acc}, ${curr.name}`;
+          }
+        }, '');
+      }
```

Comment:
We actually limited the artists to a single artist on our back end for simplicity, so we shouldn't need all of this logic.  Could probably leave it in though.  It would be an easy enough change to include all artists instead of just the first from the backend if we decide we want to spruce things up.